README.md

```diff
@@ -14,4 +14,9 @@ This is a compiler for mgol implemented in go for the final work of the compiler
 ## TODO
 
 - [ ] Implement scanner
+- [ ] Implement pattern recognition
+- [ ] Implement scanner tests
 - [x] Check why the readme is a binary file
+- [x] Implement symbol table
+- [x] Implement symbol table tests
```

Comment:
Did you implemented the symbol table on this PR?

Reply:
I did not implement the symbol table on this PR, so i will be removing this modification from the readme file.